packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -132,31 +132,51 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
     }
   };
 
+  const removeHTMLTags = (str: string) => {
+    return str.replace(/<[^>]*>/g, '');
```

Comment:
We may want to revisit this to make it more robust at some point. Right now it could cause clipping of valid text. For example, `5 < 19 <b>Yay!</b>` would have some weird behavior. Maybe it isn't too big of a deal since it is preview text?